Lib/test/test_dis.py

```diff
@@ -1050,12 +1105,12 @@ def test_explicit_first_line(self):
         self.assertEqual(list(actual), expected_opinfo_outer)
 
     def test_source_line_in_disassembly(self):
-        # Use the line in the source code
-        actual = dis.Bytecode(simple).dis()[:3]
+        # Use the line in the source code (split extracts the line no)
+        actual = dis.Bytecode(simple).dis().split(" ")[0]
```

Comment:
@syncosmic I think you can simplify the call by just stripping all surrounding whitespace rather than specifically leading spaces:

    actual = dis.Bytecode(simple).dis().strip().partition(" ")[0]

The fact there's only one string literal in there then more clearly conveys "partition the text on the first internal space".